features/step_definitions/question_steps.rb

```diff
@@ -69,8 +69,13 @@
     select '11', from: 'research_session_start_datetime_4i', visible: false
     select '30', from: 'research_session_start_datetime_5i', visible: false
   end
-  fill_in 'How long will the session be? (optional)', with: '5 minutes'
-  fill_in 'What do participants need to bring? (optional)', with: 'Nothing'
+  @session_duration = '5 minutes'
+  fill_in 'How long will the session be? (optional)', with: @session_duration
+  @session_location = 'Rockford House, Leeds'
+  save_and_open_page
```

Comment:
Ahh you were right @rgarner, we definitely need fresh eyes to look at this. Remove save_and_open_page.